samples/snippets/subscriber_test.py

```diff
@@ -208,7 +208,7 @@ def test_create_subscription_with_dead_letter_policy(
         pass
 
     subscriber.create_subscription_with_dead_letter_topic(
-        PROJECT_ID, TOPIC, SUBSCRIPTION_DLQ, DEAD_LETTER_TOPIC
+        PROJECT_ID, TOPIC, SUBSCRIPTION_DLQ, DEAD_LETTER_TOPIC, 10
```

Comment:
Question for this line and line 222
- What are the values 10 and 20? Why did we choose them to be 10 and 20 and not whatever the default is?
- If we do need to pass them and not use the default, please use a constant (or two if you want to have the separate values 10&20) like the other arguments passed in the tests - it makes for easier maintenance

Reply:
I addressed this in a seperate commit https://github.com/googleapis/python-pubsub/pull/236/commits/7ad3b6a453cbc7f5122ffd21d6ed8d808b068ac9.

I added two constants one for the default value and one for the updated one. We could call create subscription with the default value, which is what my new commit does. However, the update subscription call should be called with a new value because we are testing to see if max_delivery_attempts changes after the call.